app/src/main/res/drawable/pointer.xml

```diff
@@ -0,0 +1,10 @@
+<vector android:height="24dp" android:viewportHeight="72.0"
```

Comment:
For correctness, this also needs xml version=...

Reply:
I tried adding `version="..."` but it threw an error. Do you have an example of what it should be?

Reply:
`<?xml version="1.0" encoding="utf-8"?>`

https://github.com/mozilla-mobile/focus-android/blob/master/app/src/main/res/values/styles.xml#L1

Just open other files and copy paste. :)

Reply:
Then again, liuche has a PR where there's no version in the vector file – maybe VectorDrawables just don't use it: https://github.com/mozilla-mobile/firefox-tv/pull/534/commits/d4b0a984497e17a7eb7bd8915df30049c67f302c#diff-4ba463c20b00ee02a671dbbf7412435bR1

Reply:
Since this is auto-generated, it's probably fine to roll without it.

Reply:
Ah, I added `<?xml version="1.0" encoding="utf-8"?>` to the top. Thought you meant to add it directly to that line.